src/main/kotlin/io/zerobase/smarttracing/GraphDao.kt

```diff
@@ -53,4 +57,182 @@ class GraphDao(private val driver: Driver) {
             }
         }
     }
+
+    /**
+     * Creates an organization node inside neo4j.
+     *
+     * @param organization name of the organization that is being created.
+     * @param phone phone number for the contact for the organization.
+     * @param email email for the contact for the organization.
+     * @param contactName name of the contact for the organization.
+     * @param address address of the organization.
+     * @param testing if the organization will be doing testing.
```

Comment:
nit: if the `organization owns sites that can do testing`

Reply:
Added this